build/3-apillon-api.md

```diff
@@ -1158,3 +1158,714 @@ curl --location --request GET "https://api.apillon.io/hosting/websites/:websiteU
   </CodeGroup>
   </div>
 </div>
+
+
+## Web3 NFTs API
+
+NFTs API provides endpoints, that can be used to create non-fungible tokens in accordance with [ERC-721](https://ethereum.org/en/developers/docs/standards/tokens/erc-721/) standard.
+
+**Note:** For now API only allows creation of NFT collections with metadata uploaded to hosting server manually and providing base URI, hosting them via Apillon API may be supported later.
```

Comment:
I would change this to:

API is for creating and managing NFTs. To prepare images and metadata you can use storage API. To learn more about metadata standards you can visit: https://docs.opensea.io/docs/metadata-standards